stories/components/login/login.stories.js

```diff
@@ -37,6 +37,19 @@ export const ShowPresenceLogin = () => html`
   <mgt-login show-presence login-view="full"></mgt-login>
 `;
 
+export const RightAligned = () => html`
```

Comment:
We have the RTL story, do we need this one?

Reply:
Yes, this shows the defect that was initially reported as fixed, the RTL story does not.

They are slightly different. Here's a comparison from another branch where the layout defect is present:
RTL:
![image](https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/7122716/76b29999-eec1-405d-8c40-644c6f7d3a60)

Right Aligned:
![image](https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/7122716/dd3f6c58-6c80-4466-a237-2140516ac64d)

Reply:
That makes sense. This is good to go.